src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+                        }
+                        Todo task = new Todo(false, tmp[1]);
+                        this.taskList.add(task);
+                        this.ui.printTodo(task, this.taskList.showList().size());
+                } else if (command.startsWith("deadline")) {
+                    String[] tmp = command.split("deadline |/by ");
+                    String todo = tmp[1];
+                    String datetime = tmp[2].trim();
+                    DateTimeFormatter a = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+
+                    try {
+                        LocalDateTime date = LocalDateTime.parse(datetime, a);
+                        System.out.println(date);
+                        Deadline task = new Deadline(false, todo, date);
+                        this.taskList.add(task);
+                        this.ui.printDeadline(task, this.taskList.showList().size());
+                    } catch (DateTimeException e) {
+                        this.ui.printDeadlineError();
+                    }
+                } else if (command.startsWith("event")) {
+                    String[] tmp = command.split("event |/from |/to ");
+                    Event task = new Event(false, tmp[1], tmp[2], tmp[3]);
+                    this.taskList.add(task);
+                    this.ui.printEvent(task, this.taskList.showList().size());
+                } else if (command.startsWith("delete")) {
+                    String num = s;
+                    int target = Integer.parseInt(num);
+
+                    // Timo.Task that is deleted
+                    Task task = this.taskList.delete(target);
+                    this.ui.printDelete(task, this.taskList.showList().size());
+                } else if (command.startsWith("find")) {
+                    String phrase = command.split(" ", 2)[1];
+
+                    // TaskList to print out
+                    TaskList lst = new TaskList();
+
+                    for (Task task: this.taskList.showList()) {
+                        if (task.toString().contains(phrase)) {
+                            lst.add(task);
+                        }
+                    }
+                    this.ui.printList(lst);
+
+                } else {
+                    throw new TimoException("I'm sorry, I do not know what that means");
+                }
+        }
+    }
+}
+
+
+public class Timo {
+
+    private Storage storage;
+    private TaskList tasks;
+    private UI ui;
+
+    private Parser parser;
+
+
+    public Timo(String filepath) {
+        ui = new UI();
+        storage = new Storage(filepath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (IOException e) {
+            tasks = new TaskList();
+        }
+        parser = new Parser(ui, storage, tasks);
+    }
+
+    public void run() {
+        //welcome
+        ui.greet();
+
+        //print list initially
+        ui.printList(this.tasks);
+
+        boolean isExit = false;
```

Comment:
Good use of a boolean variable that sounds like booleans

Reply:
Thank you!